src/environments/launcher.ts

```diff
@@ -49,7 +50,7 @@ if (isBrowser && !isJest) {
 export async function getLauncherEnvironment(): Promise<
   LauncherEnvironment | undefined
 > {
-  if (isBrowser) {
+  if (isBrowser || isTauriWindow) {
```

Comment:
I might be reading this logic wrong, but it looks funny because isTauriWindow can never be true if isBrowser is not true. So it seems it would change nothing

Reply:
Hahaha! This PR is becoming a farce. You are totally right of course. When I started with the code changes I assumed there will be a difference between browser mode and tauri mode but I ended up using a solution that functions equally for both.

Reply:
hehehe